packages/angular/src/app/shared/sdk/model/inline-response-200-1.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * loopback4-example-shopping
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.1.1
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+export interface InlineResponse2001 { 
```

Comment:
To resolve the responses to meaningful model names you need to ensure that the response schemas have a title property.  The following would resolve `InlineResponse2001` to `AuthResponse`:

```ts
{
  type: 'object',
  title: 'AuthResponse',
  properties: {
    token: {
      type: 'string',
    },
  },
}
```

Reply:
I see you've done it further down! 👍